telegram/bot.py

```diff
@@ -1985,14 +2008,24 @@ def answer_pre_checkout_query(self, pre_checkout_query_id, ok, error_message=Non
             :class:`telegram.TelegramError`
 
         """
-        url = '{0]/answerPreCheckoutQuery'.format(self.base_url)
+
+        if ((ok is not True and error_message is None) or
+                (ok is True and error_message is not None)):
+            raise TelegramError(
+                'answerPreCheckoutQuery: If ok is True, there should '
+                'not be error_message; if ok is False, error_message '
+                'should not be empty')
```

Comment:
Hmm.. you seen to be correct... I don't really have a preference either way then... It's your call @tsnoam